src/blog/2022/008-drupal-integration.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Powerful rules engine in Drupal integrates with BPMN.iO
+description: This blog post explains how Drupal integrates BPMN.iO as a graphical UI for its rules engine to expose all its automation capabilities to site builders without coding requirements.
+preview_image: https://www.drupal.org/files/project-images/bpmn_io_02.png
+url: blog/posts/...
+layout: blogpost.hbs
+slug: ...
+author:
+  - Jürgen Haas<https://www.drupal.org/u/jurgenhaas>
+published_time: "..."
+modified_time: "..."
+published: ...
+releases:
+  - 'bpmn-js'
+---
+
+<p class="introduction">
+[Drupal](https://www.drupal.org) is a robust content management system, comes with sophisticated APIs for multichannel publishing, and a track record of continuous innovation. Drupal is the best digital experience platform (DXP) on the web, proudly open source.
+
+Applications build with Drupal get customized and extended by an extensive eco-system of open source modules, or with custom modules developed in PHP. With ECA, one of those modules, all aspects of Drupal can be automated and will be processed in a black box following configured models.
+
+BPMN.iO got integrated as a Drupal module to provide the UI to create, review and maintain the configured automation with BPMN models.
+</p>
+
+## ECA architecture in Drupal
+
+The Drupal module [ECA](https://www.drupal.org/project/eca) is an automated processor built around the "Event - Condition - Action" paradigm, hence the name, and operates like a black box completely in the background. Like all Drupal configuration, ECA models are managed as config entities in Yaml format as well.
+
+By design, ECA does not provide any UI to create or edit those config entities as it focusses on its role as a processor only. Instead, ECA comes with a number of APIs to allow UI tools receiving the lists of available events, conditions and actions that are available on the current Drupal site and to save back their models to ECA.
+
+As ECA is brand new and only got released on June 22, 2022 as a Release Candidate, there isn't a huge number of UI modellers available. There are just 2: BPMN.IO and Camunda, the former is fully integrated in Drupal's admin UI, the latter can be used as a desktop application communicating with the Drupal site through file for templates and the XML files.
+
+## Simple get going with BPMN.iO
+
+Like ECA as well, the [BPMN.iO](https://www.drupal.org/project/bpmn_io) module can be installed and enabled on any existing Drupal site without further dependencies. Minimum supported Drupal version is 9.4.0 though.
+
+Once enabled, site builders with appropriate permissions can start building their models without any configuration necessary:
+
+<div class="figure">
+  <img src="https://www.drupal.org/files/project-images/bpmn_io_02.png">
```

Comment:
Let's add `alt` for the images. More on that at https://supercooldesign.co.uk/blog/how-to-write-good-alt-text

Reply:
Incorporated via https://github.com/bpmn-io/bpmn.io/pull/145/commits/1ef8e8d95a2e6a05cc1ece1c44d355249470ff11.